clients/redigo.go

```diff
@@ -0,0 +1,387 @@
+package clients
+
+import (
+	"fmt"
+	"github.com/Shivam010/go-rejson/rjs"
+	redigo "github.com/gomodule/redigo/redis"
+	"strings"
+)
+
+// Redigo implements ReJSON interface for GoModule/Redigo Redis client
+// Link: https://github.com/gomodule/redigo
+type Redigo struct {
+	Conn redigo.Conn // import redigo "github.com/gomodule/redigo"
+}
+
+// JSONSet used to set a json object
+//
+// ReJSON syntax:
+// 	JSON.SET <key> <path> <json>
+// 			 [NX | XX]
+//
+func (r *Redigo) JSONSet(key string, path string, obj interface{}, opts ...rjs.SetOption) (
+	res interface{}, err error) {
```

Comment:
[Nit]: Was this a gofmt thing or did you purposely move this to the next line ?

Reply:
It's actually because of the linter `lll` which limits the length of a line to 120, hence, we are forced to move it to next line - to pass it.